tests/python/test_clang.py

```diff
@@ -499,7 +499,32 @@ def test_probe_read_nested_deref2(self):
         b = BPF(text=text)
         fn = b.load_func("test", BPF.KPROBE)
 
-    def test_probe_read_nested_deref_func(self):
+    def test_probe_read_nested_deref3(self):
+        text = """
+#include <net/inet_sock.h>
+int test(struct pt_regs *ctx, struct sock *sk) {
+    struct sock **ptr1, **ptr2 = &sk;
+    ptr1 = &sk;
+    return (*ptr1)->sk_daddr + (*ptr2)->sk_daddr;
+}
+"""
```

Comment:
Missing
```
b = BPF(text=text)
fn = b.load_func("test", BPF.KPROBE)
```
here?

Reply:
Ah, yes. I fixed it in local and forgot to commit it. I just updated the branch.